content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,22 @@
+# Getting Started with Entity Framework Core
+
+## Who is this tutorial meant for?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
```

Comment:
We probably don't need to include the "refresher course" part. Stating that the tutorial is meant for beginners will be enough to give experienced devs an idea of whether this tutorial will be useful to them.